src/main/java/org/cristalise/restapi/ItemRoot.java

```diff
@@ -142,12 +142,12 @@ public Response getMasterOutcome(
                 final Schema schema = LocalObjectLoader.getSchema(type, schemaVersion);
                 script = LocalObjectLoader.getScript(scriptName, scriptVersion);
 
-                return returnScriptResult(scriptName, item, schema, script, new CastorHashMap(), produceJSON(headers.getAcceptableMediaTypes()));
+                return scriptUtils.returnScriptResult(scriptName, item, schema, script, new CastorHashMap(), produceJSON(headers.getAcceptableMediaTypes()));

```

Comment:
Before your last commit, In which class did it find the `returnScriptResult` method?